### PR TITLE
Komiser dashboard enhancements

### PR DESCRIPTION
**Pull request by victorgaard**
## Changes Made
- Select dropdown component to match the Design System
- Cloud map tooltip has 2 updates: 
  - It displays the percentage of total
  - It has a shortcut: once a region is clicked, user is taken to the inventory page with the resources filtered by the region.
- Resources manager tooltip is also displaying the percentage of total

## Screenshots
- `Select dropdown`
image

- `Select dropdown @ Storybook`
image

- `Cloud map widget tooltip`
image

- `Resources manager widget tooltip`
image

### CODE REVIEW

File: dashboard/components/select/Select.tsx [hunk 1]
```diff
@@ -1,61 +1,93 @@
...
+          <div className="absolute top-[4.15rem] z-[21] w-full overflow-hidden rounded-lg border border-black-200 bg-white p-2 shadow-lg">
+            <div className="flex w-full flex-col gap-2">
+              {values.map((item, idx) => {
+                const isActive = value === item;
+                return (
+                  <button
+                    key={idx}
+                    className={`flex items-center justify-between rounded py-2 px-3 text-left text-sm text-black-400 hover:bg-black-150 ${
+                      isActive ? 'bg-komiser-150' : ''
+                    }`}
```
Comment (eneskaya) [inline dashboard/components/select/Select.tsx:78]: optiona/nitpickl: Using [`classnames`](https://github.com/JedWatson/classnames) package here (and everywhere else with conditional classes) could make things more readable :) e.g.

```tsx
className={classnames('flex items-center ...', { 'bg-komiser-150': isActive })}
```

**Review by mlabouardy — APPROVED** [review]

File: dashboard/components/select/Select.tsx (hunk 1 above)
Comment (victorgaard) [reply dashboard/components/select/Select.tsx:78]: Ohhh didn't know about this package, seems easier to read indeed. We do have a LOT of conditional classes in our components hehe :)

Comment (victorgaard) [reply dashboard/components/select/Select.tsx]: Installed the package and updated the Select component, will be doing more refactors soon.

Comment (eneskaya) [reply dashboard/components/select/Select.tsx]: Awesome!

**Review by eneskaya — APPROVED** [review]

**Pull request merged**